### DISCUSSION TITLE
How do I support generating fsdb wavefiles and opening waveforms with verdi

### DISCUSSION THREAD

**Discussion by jj319410** [cocotb/cocotb#3915] [Q&A] [answered]
I add “initial begin $fsdbDumpfile(); $fsdbDumpfile; end”   But Error :""
![Clip_20240603_111127](https://github.com/cocotb/cocotb/assets/2824880/58e2a03c-5ed6-4d70-a99f-fd30514be9ea)

**Comment by ktbarrett** [comment, marked as answer]
I'm guessing you pass the appropriate command line arguments to either `COMPILE_ARGS`, `SIM_ARGS`, or both. And if you want to use the waveforms viewer, just call it yourself with the file you created by passing the previous args.

**Reply by jj319410** [reply]
![Clip_20240603_140240](https://github.com/cocotb/cocotb/assets/2824880/5d015367-5fea-424b-8917-13e1b8062dee)
I add the option`-P $(LD_LIBRARY_PATH)/novas.tab $(LD_LIBRARY_PATH)/pli.a` in the COMPILE_ARGS but the error  ask me the novas.tab is not exists or readable . I ensure the novas.tab is in the path ，and can be read.

**Reply by jj319410** [reply]
I replace the path with system variables with an absolute path, and that should solve it. However, I don't know why I can't use a path with system variables.
